en/cpp/troubleshooting.md

```diff
@@ -0,0 +1,78 @@
+# Troubleshooting
+
+## User callbacks #{user_callbacks}
+
+All callbacks back to the user of the library are called from one thread. The callbacks are stored in a queue. If the user does not return quickly enough from the callback, it can happen that the user callback queue fills up.
+
+If a callback takes more than one second, the user sees the warning:
+```
+[02:56:26|Warn ] Callback took more than 1 second to run.
+See: https://mavsdk.mavlink.io/develop/en/cpp/troubleshooting.html#user_callbacks (system_impl.cpp:327)
+```
+
+At 10 queued callbacks, the user sees the warning:
+```
+[02:56:26|Warn ] User callback queue too slow.
+See: https://mavsdk.mavlink.io/develop/en/cpp/troubleshooting.html#user_callbacks (system_impl.cpp:1213)
+```
+At 100 queued callbacks, the user sees the error message:
+```
+[02:56:35|Error] User callback queue overflown
```

Comment:
@julianoes I've subedited and I'm going to merge.
But I would change: overflown > overrun